.github/workflows/source_clear_cron.yml

```diff
@@ -12,7 +14,13 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
+      - uses: actions/setup-go@v3
```

Comment:
Maybe add a `name:` attributes to each of these steps?

Reply:
I believe, the name is already there 
`- name: Source clear scan`

Reply:
Ok. All good. 

I'm just nit-picking, is all. I tend to put the names to every step, e.g. [C#](https://github.com/optimizely/csharp-sdk/blob/master/.github/workflows/csharp.yml#L39)...but I admit it's overboard.

Reply:
Ah, got it now. It is a good idea to add a name. Adding it. Thanks